x-pack/plugins/security_solution/public/detections/components/alerts_kpis/common/components.tsx

```diff
@@ -36,7 +36,7 @@ interface StackedBySelectProps {
 }
 
 export const StackByComboBoxWrapper = styled.div`
-  width: 400px;
+  max-width: 400px;
```

Comment:
this was causing the entire overview page to overflow at certain screen sizes.
Before:
<img width="813" alt="Screen Shot 2022-05-10 at 10 26 08 AM" src="https://user-images.githubusercontent.com/6935300/167677212-c0a19d66-1d6e-430e-9ad8-05ee135673f2.png">

After:
<img width="772" alt="Screen Shot 2022-05-10 at 10 25 50 AM" src="https://user-images.githubusercontent.com/6935300/167677192-d8d6d903-055f-46d4-b7e5-05b99b5cffbe.png">